code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -204,6 +204,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/bonesetter(src)
 	src.modules += new /obj/item/weapon/circular_saw(src)
 	src.modules += new /obj/item/weapon/surgicaldrill(src)
+	src.modules += new /obj/item/weapon/gripper/chemistry(src)

```

Comment:
I do not support this for the fact that it blurs the line between our two very different medical borgs.

Right now we have one borg that is specialized on surgery and has got a hypospray.

Then we have a more all-round borg that can´t do surgery, but is equipped with the items to work in the chemistry lab.

I think changing this should be discussed on the forums, as it provides a significant edge to one of the two cyborgs.
(With this change the surgical borg can do pretty much everything the crisis borg can do)